test/bdd/cmd/issuer/main.go

```diff
@@ -0,0 +1,91 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+
+	"github.com/gorilla/mux"
+	"github.com/trustbloc/edge-core/pkg/log"
+)
+
+var logger = log.New("edge-adapter/issuer")
+
+const (
+	addressPattern = ":%s"
+)
+
+func main() {
+	port := os.Getenv("ISSUER_PORT")
+	if port == "" {
+		panic("port to be passed as ENV variable")
+	}
+
+	router := mux.NewRouter().StrictSlash(true)
+	router.HandleFunc("/credential", createUserDataVCHandler).Methods(http.MethodPost)
+
+	logger.Fatalf("issuer server start error %s", http.ListenAndServe(fmt.Sprintf(addressPattern, port), router))
+}
+
+func createUserDataVCHandler(rw http.ResponseWriter, req *http.Request) {
+	msg, err := ioutil.ReadAll(req.Body)
+	if err != nil {
+		logger.Errorf("[issuer] failed to read request - err:%s", err.Error())
+		rw.WriteHeader(http.StatusBadRequest)
+	}
+
+	// TODO add validation, for now return VC
+
+	_, err = rw.Write([]byte(prCardVC))
+	if err != nil {
+		logger.Errorf("[issuer] failed to send vc - err:%s", err.Error())
+		rw.WriteHeader(http.StatusBadRequest)
+	}
+
+	rw.WriteHeader(http.StatusOK)
+
+	logger.Infof("[issuer] req: %s resp=%s", string(msg), prCardVC)
+}
+
+const (
+	prCardVC = `{
```

Comment:
why not a credit card statement credential?

Reply:
CreditCard statement context/example is not defined yet. For now, I added PRCard.